### PR TITLE
Final edits and Vale fixes in OIDC authorization code flow mechanism for protecting web applications guide for 3.2

### PR DESCRIPTION
**Pull request by sheilamjones**
This PR addresses final edits, fixes Vale errors and warnings, and pre-QE checks in the _OIDC authorization code flow mechanism for protecting web applications_ guide for 3.2

### CODE REVIEW

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc [hunk 1]
```diff
@@ -1466,19 +1485,20 @@ Additionally, `OidcWiremockTestResource` set token issuer and audience to `https
...
 
-First, prepare `application.properties`. You can start with a completely empty `application.properties` as `Dev Services for Keycloak` will register `quarkus.oidc.auth-server-url` pointing to the running test container as well as `quarkus.oidc.client-id=quarkus-app` and `quarkus.oidc.credentials.secret=secret`.
+First, prepare `application.properties`.
+You can start with a completely empty `application.properties` file as `Dev Services for Keycloak` will register `quarkus.oidc.auth-server-url` pointing to the running test container as well as `quarkus.oidc.client-id=quarkus-app` and `quarkus.oidc.credentials.secret=secret`.
 
-But if you already have all the required `quarkus-oidc` properties configured then you only need to associate `quarkus.oidc.auth-server-url` with the `prod` profile for `Dev Services for Keycloak`to start a container, for example:
+However, if you already have all the required `quarkus-oidc` properties configured, then you only need to associate `quarkus.oidc.auth-server-url` with the `prod` profile for `Dev Services for Keycloak`to start a container, for example:
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc:1494]:
```suggestion
However, if you already have all the required `quarkus-oidc` properties configured, then you only need to associate `quarkus.oidc.auth-server-url` with the `prod` profile for `Dev Services for Keycloak` to start a container, for example:
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc [hunk 2]
```diff
@@ -1466,19 +1485,20 @@ Additionally, `OidcWiremockTestResource` set token issuer and audience to `https
 [[integration-testing-keycloak-devservices]]
 === Dev services for Keycloak
 
-Using xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] is recommended for the integration testing against Keycloak.
-`Dev Services for Keycloak` will launch and initialize a test container: it will create a `quarkus` realm, a `quarkus-app` client (`secret` secret) and add `alice` (`admin` and `user` roles) and `bob` (`user` role) users, where all of these properties can be customized.
+Use xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for integration testing against Keycloak.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc:1488]: It appears to me that this sentence does not have a  message that this is a recommended way of testing. propose to revert

Comment (sberyozkin) [inline docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc:1488]:
```suggestion
Using xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] is recommended for integration testing against Keycloak.
```

File: docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc (hunk 1 above)
Comment (sheilamjones) [reply docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc:1494]: Good catch, thank you.

**Review by sberyozkin — APPROVED** [review]
Thanks @sheilamjones

**Comment by sheilamjones** [comment]
Thank you @sberyozkin.
@aloubyansky/@gsmet/@maxandersen - This PR is ready for backport to 3.2. 
Many thanks,
Sheila

**Comment by sheilamjones** [comment]
Following revised planning for 3.2.8 (and agreed with QE), this guide will not be published in Red Hat build of Quarkus for 3.2.8. Therefore, a backport to 3.2 is not needed and I removed the backport triage labels . The fixes applied in this PR will be fixed and published in Main only.

**Pull request merged**